paths/webhooks.yaml

```diff
@@ -8,7 +8,12 @@
           content:
             application/json:
               schema:
-                $ref: ../schemas/webhooks/webhooks_list.yaml
+                type: object
+                properties:
+                  webhooks:
+                    type: array
+                    items:
+                      $ref: ../schemas/webhooks/webhook.yaml
```

Comment:
This changes allows to have one less object generated? Or is there another reason?